packages/ses/src/commons.js

```diff
@@ -294,3 +294,58 @@ export const noEvalEvaluate = () => {
   // See https://github.com/endojs/endo/blob/master/packages/ses/error-codes/SES_NO_EVAL.md
   throw TypeError('Cannot eval with evalTaming set to "noEval" (SES_NO_EVAL)');
 };
+
+// ////////////////// FERAL_STACK_GETTER FERAL_STACK_SETTER ////////////////////
+
+const er1StackDesc = getOwnPropertyDescriptor(Error('er1'), 'stack');
+const er2StackDesc = getOwnPropertyDescriptor(TypeError('er2'), 'stack');
```

Comment:
Why use 2 different error types?

Reply:
More robust sanity test, to see that the getter is the same across two different ones. Even slightly more robust that testing that two different subclasses of Error agree. 

Resolving.